content/tutorial/02-sveltekit/02-routing/07-params/README.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Route Parameters
```

Comment:
If we do the blog example, we should probably move this section above 'loading data' since it'll need parameters. I think we can make this section work without a `load` function — we can just show that if you have a `/blog/[slug]` route it will respond to any `/blog/*` pathname, and that leads naturally to talking about loading data in the next section

```suggestion
title: Route parameters
```